src/client/index.ts

```diff
@@ -30,23 +30,84 @@ export class Client {
   /**
    * Cancel an ongoing workflow
    *
+   * There are multiple ways you can cancel workflows:
+   * - pass one or more workflow run ids to cancel them
+   * - pass a workflow url to cancel all runs starting with this url
+   * - don't pass any options. in this case, all workflow will be canceled
+   *
+   * ### Cancel a set of workflow runs
+   *
    * ```ts
-   * import { Client } from "@upstash/workflow";
+   * // cancel a single workflow
+   * await client.cancel({ ids: "<WORKFLOW_RUN_ID>" })
    *
-   * const client = new Client({ token: "<QSTASH_TOKEN>" })
-   * await client.cancel({ workflowRunId: "<WORKFLOW_RUN_ID>" })
+   * // cancel a set of workflow runs
+   * await client.cancel({ ids: [
+   *   "<WORKFLOW_RUN_ID_1>",
+   *   "<WORKFLOW_RUN_ID_2>",
+   * ]})
+   * ```
+   *
+   * ### Cancel workflows starting with a url
+   *
+   * If you have an endpoint called `https://your-endpoint.com` and you
+   * want to cancel all workflow runs on it, you can use `urlStartingWith`.
+   *
+   * Note that this will cancel workflows in all endpoints under
+   * `https://your-endpoint.com`.
+   *
+   * ```ts
+   * await client.cancel({ urlStartingWith: "https://your-endpoint.com" })
+   * ```
+   *
+   * ### Cancel *all* workflows
+   *
+   * If you want to cancel all workflows under your user, you can
```

Comment:
"If you'd like to cancel all your pending and currently running workflows, you can..."